Gateway/Request/MotoRefundDataBuilder.php

```diff
@@ -0,0 +1,271 @@
+<?php
+/**
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2022 Adyen N.V. (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+
+namespace Adyen\Payment\Gateway\Request;
+
+use Adyen\Payment\Helper\ChargedCurrency;
+use Magento\Payment\Gateway\Request\BuilderInterface;
+
+/**
+ * Class MotoRefundDataBuilder
+ */
+class MotoRefundDataBuilder implements BuilderInterface
```

Comment:
Is the logic in this class also being used for non moto refunds? If so I would suggest to create a ticket to centralize it and remove the duplicated logic